roomranger/src/main/resources/templates/rooms/detail.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
Hey Luna, this is the code of my homepage! in your room detail you only need the room number, the type and the availability. everything else will be in my homepage with all the assigned rooms!

Reply:
Yeah, we will not use templates since using react now. Those all templates will be deleted at last.